packages/docusaurus-init/src/index.ts

```diff
@@ -37,19 +37,42 @@ async function updatePkg(
   await fs.outputFile(pkgPath, JSON.stringify(newPkg, null, 2));
 }
 
+function getTemplates(): {[key: string]: string} {
+  const templatesDirs = [
+    path.resolve(__dirname, '../templates'),
+    path.resolve(process.cwd(), './templates'),
+  ];
+
+  const templates = {};
+
+  for (const templatesDir of templatesDirs) {
+    if (fs.existsSync(templatesDir)) {
+      Object.assign(
+        templates,
+        ...fs
+          .readdirSync(templatesDir)
+          .filter((d) => !d.startsWith('.') && !d.startsWith('README'))
```

Comment:
Hey, good idea!

I'm curious, Why do you have a filter for directories that starts with readme? `!d.startsWith('README')`?

maybe would be good add a option in the `docusaurus.config.js` to not restrict only to the current directory.

Also in case of approved, it was good have a section in the documentation https://v2.docusaurus.io/docs/next/installation#scaffold-project-website

Reply:
@fanny thanks! Appreciate your feedback.

`readdirSync` lists all directories and files in a folder (including `README` and `.`). That condition is in place to make sure directories are only added to the probe list. Also, I didn't add that condition in my PR, that condition was already in place. I just refactored it into a separate function.

Sounds good! I'm happy to update the docs as well.